### ISSUE TITLE
Display customizations

### ISSUE THREAD

**Issue by pedropalmero** [roeybiran/finbar-issues#21]
Hi,

I'm using an ultrawide monitor and it would be great if the font size could be customizable, so it is more readable at first glance.

Maybe a preference to choose between various fixed sized (regular, large, extralarge, ...) or select an exact font size.

Also, I think it would be great if the right/left arrow keys could be used to expand/collapse the menu items without having to use the mouse or press enter and lose the current context.

Thanks a lot!

**Comment by roeybiran** [comment]
Hi there!

1. Finbar design leans on “command palette” style apps, like Spotlight, LaunchBar and Raycast. Those apps usually don’t offer such customizations, but that doesn’t mean I won’t consider adding that. How do you find working with apps such as Spotlight on your ultra wide monitor?
2. This is a known issue that will be resolved in the next update.

**Comment by pedropalmero** [comment]
Hi, thanks for answering

> Finbar design leans on “command palette” style apps, like Spotlight, LaunchBar and Raycast. Those apps usually don’t offer such customizations, but that doesn’t mean I won’t consider adding that. How do you find working with apps such as Spotlight on your ultra wide monitor?

I use Alfred too, and it allows different themes, I use the Alfred Modern theme, which is bundled by default, and works very good for me. You can see the difference.

<img width="764" alt="SCR-20241022-qmsd" src="https://github.com/user-attachments/assets/9e3e6401-f0be-4294-9949-671e9fa49350">

I attach an screenshot of finbar for comparison

<img width="1202" alt="SCR-20241022-qphq" src="https://github.com/user-attachments/assets/9b42e487-c581-450c-8263-8f15c1235351">


Maybe it is not making the font larger exactly and it is a mix of things like changing the font weight, font size or even the font family, using different icons, ..., but I find a bit difficult to read and distinguish the elements sometimes.


> This is a known issue that will be resolved in the next update.

Great!

**Comment by pedropalmero** [comment]
I was taking a look at Raycast and LaunchBar, as I have never used them, and they offer different ways to customize their aspect too.

In Raycast, there is a setting to use a larger font size or a regular one.


![image](https://github.com/user-attachments/assets/20f1dfea-1d2e-4bb3-ac8d-79b27fed6b21)


> Raycast supports light and dark themes. By default, it adopts your system appearance, however, you can select either as your preference.
>
> You can also toggle between our default Text Size, or a slightly bigger option.

I think it is an option than can suit not only for people like me, that uses a large monitor, but to people which has issues with their eyesight.


In LaunchBar, I've seen they offer to customize the font size in the search results with a setting.

Both apps, Raycast and LaunchBar support theming, which I suppose will allow to change their interface extensively, but I think that it would be out of scope.

**Comment by roeybiran** [comment]
You’re totally right, and I had completely missed those customization option in the respective apps even though I looked for those specifically. I’ll definitely add interface customization options to the roadmap! (I think prefer adding wholesale theming options rather than just text size setting, that’d probably cater to a lot more people).
Stay tuned!

**Comment by pedropalmero** [comment]
Great, thanks a lot!

**Comment by roeybiran** [comment]
Sorry it took a while! Starting from version 1.16 of the app, you can now increase text size.
